crates/contracts/src/bin/vendor.rs

```diff
@@ -55,6 +55,10 @@ fn run() -> Result<()> {
             "balancer-labs/balancer-v2-monorepo/ad1442113b26ec22081c2047e2ec95355a7f12ba/\
              pkg/deployments/tasks/20210624-stable-pool/abi/StablePoolFactory.json",
         )?
+        .github(
+            "CoWSwapEthFlow",
+            "cowprotocol/ethflowcontract/hardhatversion/build/artifacts/src/CoWSwapEthFlow.sol/CoWSwapEthFlow.json"
```

Comment:
I am depending on a hardhat build until ethcontract-rs and foundry are compatible.